src/views/portfolio/tags/TagList.vue

```diff
@@ -28,16 +28,32 @@ export default {
     PortfolioWidgetRow,
   },
   methods: {
+    deleteTags: function (tagNames) {
+      return this.axios.delete(`${this.$api.BASE_URL}/${this.$api.URL_TAG}`, {
+        data: tagNames,
+      });
+    },
     refreshTable: function () {
       this.$refs.table.refresh({
         url: `${this.$api.BASE_URL}/${this.$api.URL_TAG}`,
         silent: true,
       });
     },
   },
+  mounted() {
+    this.$refs.table.refreshOptions({
+      showBtnDeleteSelected: this.isPermitted(this.PERMISSIONS.TAG_MANAGEMENT),
```

Comment:
This should check for both `TAG_MANAGEMENT` and `TAG_MANAGEMENT_DELETE` [in accordance with the REST endpoint](https://github.com/DependencyTrack/hyades-apiserver/blob/f060f75a6aff32d88a0bd389ec2c5768989d7786/src/main/java/org/dependencytrack/resources/v1/TagResource.java#L130).